app/models/metric/capture.rb

```diff
@@ -128,32 +127,35 @@ def self.perf_capture_health_check(zone)
   end
   private_class_method :perf_capture_health_check
 
+  # Collect realtime targets and group them by their rollup parent
+  #
+  # 1. Only display rollups for Hosts
+  # 2. Since perf_target_to_interval_name(Host) == "realtime", no need to filtering on interval_name == "realtime"
```

Comment:
Is this comment important, given that the logic you're changing here removes that? It's almost more confusing to keep after the fact, if I'm understandingly correctly.